py_zipkin/zipkin.py

```diff
@@ -143,6 +154,17 @@ def __init__(
         if self.sample_rate is not None and not (0.0 <= self.sample_rate <= 100.0):
             raise ZipkinError('Sample rate must be between 0.0 and 100.0')
 
+        if not set(include).issubset(set(STANDARD_ANNOTATIONS.keys())):
+            raise ZipkinError(
+                'Only %s are supported as annotations' %
+                STANDARD_ANNOTATIONS.keys()
```

Comment:
If you do the above comment on moving the keys set generation to global scope you can replace this.